.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    ...require("@thesis-co/prettier-config"),
```

Comment:
As you [can see](https://github.com/thesis/prettier-config/blob/main/index.json) the only rule set in this config is `semi` with `false` value.
This rule is already defined by the [default Prettier's config](https://prettier.io/docs/en/configuration.html#basic-configuration) so it seems to be redundant. Unless it's planned to extend Thesis config with some additional rules?

Reply:
Our config precedes the default swap for prettier (which happened in this latest major version bump).

Let's light our custom prettier config on fire 💪